pyproject.toml

```diff
@@ -50,6 +45,6 @@ commands = pytest {posargs} tests/
 basepython = python3.8
 changedir = {toxinidir}
 commands =
-    black --check setup.py fx_crash_sig tests
-    ruff setup.py fx_crash_sig tests
+    ruff format setup.py fx_crash_sig tests
+    ruff check setup.py fx_crash_sig tests
```

Comment:
I like the new `ruff format` thing with 0.1.8. It didn't create any changes here. It's nice. One less dependency plus it re-uses all the ruff options so we don't have to do things like specify line-length and target python twice.

Reply:
Is `ruff` going to replace `black` across all the crash ingestion repos?